include/umf/proxy_lib_new_delete.h

```diff
@@ -48,6 +48,16 @@ SOFTWARE.
 #include <stdlib.h>
 #endif // _WIN32
 
+// disable warning C28196: The requirement that '(_Param_(1)>0)?(return!=0):(1)'
+// is not satisfied. (The expression does not evaluate to true.)
+// disable warning C6387: 'return' could be '0': this does not adhere to the
+// specification for the function 'new'.
```

Comment:
well this is a bug - we should check if malloc return null and throw exception in thiscase

Reply:
fixed